File: ClientApp/src/app/Models/doctor.ts
import { EntityBase } from './entity-base';


export class Doctor extends EntityBase {
  FirstName: string;
  LastName: string;
  Age: number;
  Image: number;
  Description: string;



  get fullName(): string {
     return `${this.FirstName} ${this.LastName}`;

  }
}

File: Server/DentistSite/DentistSite.Domain/Entities/Doctor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DentistSite.Domain.Entities.Base;

namespace DentistSite.Domain.Entities
{
  public class Doctor:EntityBase
  {
    public string FirstName { get; set; }

    public string LastName { get; set; }

    public int Age { get; set; }

    public byte [] Image { get; set; }

    public string Description { get; set; }
  }
}
